{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:python_bugzilla_project:python-bugzilla:0.6.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python_bugzilla_project:python-bugzilla:0.6.1:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python_bugzilla_project:python-bugzilla:0.6.2:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python_bugzilla_project:python-bugzilla:0.7.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python_bugzilla_project:python-bugzilla:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "0.8.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:17:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:18:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:opensuse:opensuse:11.4:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:opensuse:opensuse:12.2:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:opensuse:opensuse:12.3:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2013-2191"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "python-bugzilla",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "0.6.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "0.6.1"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "0.6.2"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "0.7.0"
                      },
                      {
                        "version_affected": "\u003c=",
                        "version_value": "0.8.0"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "python_bugzilla_project"
          },
          {
            "product": {
              "product_data": [
                {
                  "product_name": "fedora",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "17"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "18"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "fedoraproject"
          },
          {
            "product": {
              "product_data": [
                {
                  "product_name": "opensuse",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "11.4"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "12.2"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "12.3"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "opensuse"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "python-bugzilla before 0.9.0 does not validate X.509 certificates, which allows man-in-the-middle attackers to spoof Bugzilla servers via a crafted certificate."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-20"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "openSUSE-SU-2013:1154",
          "refsource": "SUSE",
          "tags": [],
          "url": "http://lists.opensuse.org/opensuse-updates/2013-07/msg00025.html"
        },
        {
          "name": "openSUSE-SU-2013:1155",
          "refsource": "SUSE",
          "tags": [],
          "url": "http://lists.opensuse.org/opensuse-updates/2013-07/msg00026.html"
        },
        {
          "name": "[oss-security] 20130619 [CVE identifier assignment notification] CVE-2013-2191  python-bugzilla: Does not verify Bugzilla server certificate",
          "refsource": "MLIST",
          "tags": [],
          "url": "http://www.openwall.com/lists/oss-security/2013/06/19/6"
        },
        {
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=951594",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=951594"
        },
        {
          "name": "https://git.fedorahosted.org/cgit/python-bugzilla.git/commit/?id=a782282ee479ba4cc1b8b1d89700ac630ba83eef",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ],
          "url": "https://git.fedorahosted.org/cgit/python-bugzilla.git/commit/?id=a782282ee479ba4cc1b8b1d89700ac630ba83eef"
        },
        {
          "name": "[python-bugzilla] 20130619 ANNOUNCE: python-bugzilla 0.9.0 released",
          "refsource": "MLIST",
          "tags": [],
          "url": "https://lists.fedorahosted.org/pipermail/python-bugzilla/2013-June/000104.html"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "MEDIUM",
        "accessVector": "NETWORK",
        "authentication": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 4.3,
        "confidentialityImpact": "NONE",
        "integrityImpact": "PARTIAL",
        "vectorString": "AV:N/AC:M/Au:N/C:N/I:P/A:N",
        "version": "2.0"
      },
      "exploitabilityScore": 8.6,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "MEDIUM",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2018-10-30T16:27Z",
  "publishedDate": "2014-02-08T00:55Z"
}